packages/nc-gui/components/smartsheet-toolbar/ViewActions.vue

```diff
@@ -221,6 +219,8 @@ const { isSqlView } = useSmartsheetStoreOrThrow()
 
     <WebhookDrawer v-if="showWebhookDrawer" v-model="showWebhookDrawer" />
 
+    <SmartsheetToolbarErd v-model:modelValue="showErd" />
```

Comment:
`v-model:modelValue` is kinda unnecessary because `modelValue` is the default for v-model.
So `v-model` by itself is enough here :)